inc/composer/class-command.php

```diff
@@ -0,0 +1,195 @@
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Chassis command for Composer.
+ */
+class Command extends BaseCommand {
+	/**
+	 * Configure the command.
+	 */
+	protected function configure() {
+		$this->setName( 'dev-tools' );
+		$this->setDescription( 'Developer tools' );
+		$this->setDefinition( [
+			new InputArgument( 'subcommand', InputArgument::REQUIRED, 'phpunit, scaffold' ),
+			new InputOption( 'chassis', null, null, 'Run commands in the Local Chassis environment' ),
+			new InputArgument( 'options', InputArgument::IS_ARRAY ),
+		] );
+		$this->setHelp(
+			<<<EOT
+Run a dev tools feature.
+
+To run PHPUnit integration tests:
+    phpunit [--chassis] [--] [options]
+                                use `--` to separate arguments you want to
+                                pass to phpunit. Use the --chassis option
+                                if you are running Local Chassis.
+Create scaffolding for various tools:
+    scaffold <type>             possible values are 'phpunit' and 'travis'
+EOT
+		);
+	}
+
+	/**
+	 * Wrapper command to dispatch subcommands
+	 *
+	 * @param InputInterface $input
+	 * @param OutputInterface $output
+	 * @return int Status code to return
+	 */
+	protected function execute( InputInterface $input, OutputInterface $output ) {
+		$subcommand = $input->getArgument( 'subcommand' );
+		switch ( $subcommand ) {
+			case 'phpunit':
+				return $this->run_command( $input, $output, 'vendor/bin/phpunit' );
+
+			case 'scaffold':
+				return $this->scaffold( $input, $output );
+
+			default:
+				throw new CommandNotFoundException( sprintf( 'Subcommand "%s" is not defined.', $subcommand ) );
+		}
+	}
+
+	/**
+	 * Copies boilerplate files over to the project root.
+	 *
+	 * @param InputInterface $input
+	 * @param OutputInterface $output
+	 * @return int
+	 */
+	protected function scaffold( InputInterface $input, OutputInterface $output ) {
+		// Scaffold PHPUnit by default.
+		$target = $input->getArgument( 'options' )[0] ?? null;
+
+		switch ( $target ) {
+			case 'phpunit':
+				return $this->scaffold_phpunit( $input, $output );
+
+			case 'travis':
+				return $this->scaffold_travis( $input, $output );
+
+			default:
+				throw new CommandNotFoundException( sprintf(
+					'"%s" is not a recognised scaffold type, it must be one of "phpunit" or "travis".',
+					$target
+				) );
+		}
+	}
+
+	protected function scaffold_phpunit( InputInterface $input, OutputInterface $output ) {
+		$package_root = dirname( __DIR__, 2 );
+
+		// Check for an existing tests directory.
+		if ( file_exists( $this->get_root_dir() . '/.tests' ) ) {
+			$output->writeln( '<error>You already have a ".tests" directory present in your project!</>' );
+			$output->writeln( '<comment>If you want to compare the recommended Altis testing setup you can temporarily rename your current tests directory and run this command again.</>' );
+			return 1;
+		}
+
+		if ( file_exists( $this->get_root_dir() . '/phpunit.xml.dist' ) ) {
```

Comment:
Could we not put this in root, and sepcify the config path with `-c` on `phpunit`'s invocation? I think it would be better to not support a custom phpunit _as a start_.